lighthouse-core/config/default-config.js

```diff
@@ -505,23 +515,25 @@ const defaultConfig = {
     'best-practices': {
       title: str_(UIStrings.bestPracticesCategoryTitle),
       auditRefs: [
-        {id: 'appcache-manifest', weight: 1},
-        {id: 'is-on-https', weight: 1},
-        {id: 'uses-http2', weight: 1},
-        {id: 'uses-passive-event-listeners', weight: 1},
-        {id: 'no-document-write', weight: 1},
-        {id: 'external-anchors-use-rel-noopener', weight: 1},
-        {id: 'geolocation-on-start', weight: 1},
-        {id: 'doctype', weight: 1},
-        {id: 'charset', weight: 1},
-        {id: 'no-vulnerable-libraries', weight: 1},
-        {id: 'js-libraries', weight: 0},
-        {id: 'notification-on-start', weight: 1},
-        {id: 'deprecations', weight: 1},
-        {id: 'password-inputs-can-be-pasted-into', weight: 1},
-        {id: 'errors-in-console', weight: 1},
-        {id: 'image-aspect-ratio', weight: 1},
-        {id: 'image-size-responsive', weight: 1},
+        // Trust & Safety
+        {id: 'is-on-https', weight: 1, group: 'trust-and-safety'},
+        {id: 'external-anchors-use-rel-noopener', weight: 1, group: 'trust-and-safety'},
+        {id: 'geolocation-on-start', weight: 1, group: 'trust-and-safety'},
+        {id: 'notification-on-start', weight: 1, group: 'trust-and-safety'},
+        {id: 'no-vulnerable-libraries', weight: 1, group: 'trust-and-safety'},
+        // General Group
```

Comment:
how do we feel about grouping a little further team?

I could see a few potential collections if we wanted to go down that road

- Performance (now that I think about it maybe these should be just in diagnostics if it's about performance but we can't measure it yet, also why haven't we made uses-http2 an opportunity yet 😂?)
  - uses-http2
  - no-document-write
  - uses-passive-event-listeners
  - charset
- UX
  - image-aspect-ratio
  - image-size-responsive
  - password-inputs-can-be-pasted-into
- Code Health
  - errors-in-console
  - deprecations
  - appcache-manifest (basically just an extremely specific deprecation)
- Browser Compat (more of a forward thinking category)
  - doctype
- General
  - js-libraries (actually feels more like a diagnostic the way it presents itself right now...)


I can turn this into an issue if we're interested in discussing this route.

Reply:
👍  on this taxonomy. 
👍 on moving the perf ones to be perf category diagnostics.

Reply:
I was thinking of adding a UX category too and I like your other suggestions. I also share your sentiment about performances best practices making sense as diagnostics.

Reply:
cool beans! are you OK with doing those moves in this same PR since it's all just group munging @Beytoven? I think the final set we agree on looks like

- Move to performance diagnostics
  - uses-http2
  - no-document-write
  - uses-passive-event-listeners
- UX
  - image-aspect-ratio
  - image-size-responsive
  - password-inputs-can-be-pasted-into
- Browser Compatibility
  - doctype
  - charset
- General
  - js-libraries
  - errors-in-console
  - deprecations
  - appcache-manifest


so only two new categories

Reply:
Looks good. I can do it as part of this PR.